Notes:

- Run the code in the same folder as needed input files:
    - `python format_metadata_allencell.py`
    - It expects `allencell.metadata.csv` or `opencell.metadata.csv` that are generated by our processing pipeline.
    - Other input files are included in this folder. See comments in code and below for where these input files come from.

- For AllenCell, to match images to gene name and Ensembl ID
    - Image metadata (including gene names) is in the `manifest.csv` file from https://github.com/AllenCell/cvapipe_analysis download. `csvcut -c 1,3,9 manifest.csv > manifest_IDs.csv`
    - Obtain conversion table from https://www.ensembl.org/biomart/martview:
         - Ensembl Genes 112, Human genes (GRCh38.p14)
         - Attributes:Gene stable ID, Gene name, RefSeq match transcript (MANE Select)
         - Note this table contains lots of duplicate rows
    - `CETN2`, `FBL`, `TJP1` match to multiple Ensembl ID due to having multiple alleles. Keep the ones that match to RefSeq transcript ID on the https://www.allencell.org/cell-catalog.html details page.
    - Replaced gene name `HIST1H2BJ` with HGNC name `H2BC11`. See https://www.allencell.org/cell-catalog.html line 61 and https://www.ncbi.nlm.nih.gov/gene/8970
    - `AAVS1` is a viral insertion site that does not have Ensembl ID.
      
